README.md

```diff
@@ -18,11 +29,126 @@ Current state of the software:
 
 For descriptions of the models supported in our library, discussion of software design, and examples, please refer to the following paper: https://arxiv.org/abs/2205.08144
 
+- Two lightweight interfaces are available for `Python` ([BayesMixPy](https://github.com/bayesmix-dev/bayesmix/tree/master/python#readme)) and `R` (BayesMixR)
+<!-- Add link after merge !!! -->
+
 # Installation
 
+Compiling and building `bayesmix` requires a modern C++ compiler, `git`, the GNU `make` utility, `cmake` (version >= 3.20) and `pkg-config`. Make these packages available varies between operating systems.
+
+**Warning**: Check the version of `cmake` available in your package manager default repository. You may need to install a more recent version in another way. If this is your case, please go to the [official CMake GitHub repository](https://github.com/Kitware/CMake) and follow the installation instructions.
+
+## Requirements - Linux
+
+On Linux, the required packages can be installed via the default package manager. For instance, in Ubuntu it is sufficient to run
```

Comment:
I would phrase this more explicitly, something like:
"On Linux systems, the required packages can be installed via the system's default package manager. For instance, in Ubuntu or other Debian-based distros it is..."